addons/product/static/src/product_catalog/order_line/order_line.js

```diff
@@ -28,7 +28,13 @@ export class ProductCatalogSOL extends Component {
     }
 
     get price() {
-        return formatMonetary(this.props.price, { currencyId: this.env.currencyId, digits: this.env.digits });
+        const { currencyId, digits } = this.env;
+        return formatMonetary(this.props.price, { currencyId, digits });
     }
 
+    get quantity() {
+        const digits = [false, this.env.precision];
+        const options = { digits, decimalPoint: ".", thousandsSep: "" };
```

Comment:
Shouldn't the options be dependent on the user's language? I.e. it looks like formatFloat should already be doing it for you? Or maybe I'm misunderstanding the point of this

Reply:
Since the value is displayed in an input, I'm not sure but it seems safer to avoid thousand separation and always use the dot as the decimal point